tools/icongen.config.js

```diff
@@ -87,6 +87,8 @@ module.exports = {
     'ic_unfold_less_24px.svg',
     'ic_fullscreen_24px.svg',
     'ic_fullscreen_exit_24px.svg',
+    'ic_power_settings_new_24px.svg',
+    'ic_language_24px.svg'
```

Comment:
Please add a trailing comma (reason: next one who adds something does not need to change this particular line; or even think about adding one here). Thanks!